test/js/ui/map.test.js

```diff
@@ -646,6 +647,50 @@ test('Map', (t) => {
         t.end();
     });
 
+    t.test('#querySourceFeatures', (t) => {
```

Comment:
`querySourceFeatures` tests are currently split between [`tile.test.js`](http://github.com/mapbox/mapbox-gl-js/blob/0088929dcc934d1f41d5aec5c5bfe9bdd165f89f/test/js/source/tile.test.js#L38-L39) and [`style.test.js`](http://github.com/mapbox/mapbox-gl-js/blob/0088929dcc934d1f41d5aec5c5bfe9bdd165f89f/test/js/style/style.test.js#L1179-L1180). Could you move this test to one of those two places?